module-3/web-app/src/components/Pages/Documents/Documents.jsx

```diff
@@ -1,10 +1,30 @@
-import React from "react"
+import React, {useState, useEffect} from "react"
+import {useSearchParams} from "react-router-dom"
+import style from "./Documents.module.scss"
 
+//TODO in progress documents catalogue page
 const Documents = () => {
+    const [documents, setDocuments] = useState('')
+    const [searchParams, setSearchParams] = useSearchParams()
+    const projectId = searchParams.get('projectid')
+   
+    useEffect(() => {
```

Comment:
Add `TODO` comment to move this in component fetch to a redux action later on

Reply:
Done